tests/next_tests/integration_tests/feature_tests/iterator_tests/test_builtins.py

```diff
@@ -363,4 +363,4 @@ def sten_cast(value):
     )
 
     if validate:
-        assert out[0] == expected_value
+        assert out.ndarray[0] == expected_value
```

Comment:
This is an example where I think it makes sense to extract the low-level buffer.

Reply:
np.asarray()